src/plugins/inputs/away.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+exports.commands = ["away", "back"];
+
+exports.input = function(network, chan, cmd, args) {
+	if (cmd === "away") {
+		let reason = " ";
+
+		if (args.length > 0) {
```

Comment:
> I'm very against it because the standard is that "/away" sets you as unaway. I am very strongly for standards, why have a standard if we are going to ignore it?

Not exactly. While I am myself very much into standardization (just look at my resume lol), this is a protocol, not a standard. The former ensures machine-to-machine compatibility while the latter is more often designed for humans.
In this case, we are talking about a *de facto* standard, a convention. And the reason for this debate is because we are seeing 2 of these: one with `/away [message]`/`/back` and one with `/away message`/`/away`.
Similarly, if we really wanted The Lounge to talk the same language than the protocol, we would use `AWAY` and not `/away` but starting with a slash is a convention so well spread that it makes sense to do it. (As far as I know, starting a command with a slash is not part of the protocol). We also send private messages without starting with `PRIVMSG`. While it would be terrible to do anything else, that's still a difference between client usage and protocol.

> I do agree with you on the UX side of things, but we shouldn't just unilaterally ignore the standard because we don't agree with it.

We're not ignoring the protocol nor the conventions. If there were no other consequences or if `/away` setting you away was not used by any other IRC clients out there, I'd agree with you. But to me, and that's the main reason why I'm in favor of @xPaw's solution, typing `/away` to remove the away status is completely counter-intuitive and I would not be surprised if that was why so many other clients are going with the same solution.

In short, I definitely agree with you that following the protocol and conforming to standards are very important but we are not ignoring them right now. However, for a client designed to humans, proper UX should not be affected by these.